ocs_ci/utility/multicluster.py

```
@@ -303,9 +311,254 @@ def reeval_upgrade_order(self, phase_order, zrank, role_rank):
+
+    # Maps each phase-order constant -> scale exponent.
+    # Higher scale == later in the overall test run.
+    UPGRADE_TEST_ORDER = {
+        constants.ORDER_OCP_UPGRADE: 1,
+        constants.ORDER_MCE_UPGRADE: 2,
+        constants.ORDER_OCP_ON_KUBEVIRT_UPGRADE: 3,
+        constants.ORDER_OCS_UPGRADE: 4,
+        constants.ORDER_MCO_UPGRADE: 5,
+        constants.ORDER_DR_HUB_UPGRADE: 6,
+        constants.ORDER_ACM_UPGRADE: 7,
+    }
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)